NewForce-Cohort-6/tabloidmvc-the-tootsie-rolls
# CN-CreateAComment#20

# Description

Added functionality to add a comment to a post. Added `Add Comment` button  on post details page and a view with a form to submit a comment that redirects to the post's comment list page.


# Testing Instructions

1. git add and git commit whatever you're working on on your branch
2. git checkout main
3. git fetch --a
4. git checkout `CN-CreateAComment#20`
5. If you don't see the most recent version of my files, git pull origin `CN-CreateAComment#20`
6. If you haven't created the Tabloid MVC database, please do so.
7. Use this data to fill out tables and columns: [SQL Tables](https://github.com/NewForce-Cohort-6/tabloidmvc-the-tootsie-rolls/blob/main/SQL/01_TabloidMVC_Create_DB.sql)
8. Use this to seed the database : [SQL SEED DATA](https://github.com/NewForce-Cohort-6/tabloidmvc-the-tootsie-rolls/blob/main/SQL/02_TabloidMVC_Seed_Data.sql)
9. Next seed some comment data [from here](https://github.com/NewForce-Cohort-6/tabloidmvc-the-tootsie-rolls/pull/12) if you haven't already
10. Click the execute with debugger solid green button to launch the app
11. Log into the website using “[admin@example.com](mailto:admin@example.com)” if greeted with a login page
12. From the home page, click the “POST” option in the navbar
13. Click on the eye icon next to the first post `C# is the Best Language`
14. On the details page, click `Add Comment`
15. Enter a subject and content and click Create. You should be redirected to the comment list for that post with your new comment on top
16. Click `Back to Post`
17. Click `View Comments`
18. Click `Create New`
15. Enter a subject and content and click Create. You should be redirected to the comment list for that post with your new comment on top





# Checklist:
